services/web-app/pages/designing/get-started.mdx

```diff
@@ -180,8 +178,6 @@ design kits include elements, core components, wireframes, and guidelines.
 Additional design kits compliment and extend Carbon. For example, IBM.com provides components for an
 editorial approach. More kits can be found in the [design kit catalog]().
 
-### Local design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
   designTools={['Figma']}
```

Comment:
Im step 4, delete duplicate 'Home base for the IBM Brand story, visual brand elements, guidelines, and assets.'